src/MSBuild/app.amd64.config

```diff
@@ -45,11 +45,6 @@
           <bindingRedirect oldVersion="0.0.0.0-99.9.9.9" newVersion="15.1.0.0" />
           <codeBase version="15.1.0.0" href="..\Microsoft.Build.Engine.dll"/>
         </dependentAssembly>
-        <dependentAssembly>
-          <assemblyIdentity name="Microsoft.Build.Conversion.Core" culture="neutral" publicKeyToken="b03f5f7f11d50a3a" />
-          <bindingRedirect oldVersion="0.0.0.0-99.9.9.9" newVersion="15.1.0.0" />
-          <codeBase version="15.1.0.0" href="..\Microsoft.Build.Conversion.Core.dll"/>
```

Comment:
We'll need the codebase entry in the leftover item for `Microsoft.Build.Conversion.Core`

Reply:
You'd actually already added that in #7472, and I avoided a merge conflict here. I switched which I deleted (and accepted the merge conflict) to make that clearer.